src/js/modules/Hours.js

```diff
@@ -1,11 +1,19 @@
 import React, {Component} from 'react';
 import moment from 'moment';
+import {sortBy, findIndex} from 'lodash';
 
 class Hours extends Component {
 
-  render() {
+  sort() {
+    // TODO: Joplin data MUST include data for all 7 days of week.
```

Comment:
I'm absolutely cool with leaving in TODO comments like this, but just want to make sure there is an Issue created in the Joplin repo for this.

Reply:
Yep! https://github.com/cityofaustin/joplin/issues/27